jrcc-access-spring-boot-autoconfigure/src/main/java/ca/bc/gov/open/jrccaccess/autoconfigure/plugins/sftp/AutoConfiguration.java

```diff
@@ -68,13 +68,13 @@ public AutoConfiguration(SftpInputProperties sftpInputProperties) {
         }
         boolean isAllowUnknownKeys = properties.isAllowUnknownKeys();
         factory.setAllowUnknownKeys(isAllowUnknownKeys);
-        if(!isAllowUnknownKeys){
+        if (!isAllowUnknownKeys) {
             String knownHostFileStr = properties.getKnownHostFile();
-            if(knownHostFileStr == null || knownHostFileStr.equals("") )
+            if (knownHostFileStr == null || knownHostFileStr.equals(""))
```

Comment:
can we use `StringUtils.isBlank()` here to simplify this check perhaps?
https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isBlank-java.lang.CharSequence-

Reply:
good call, since we have lang3, let's use it

Reply:
yes. good catch.
Changed.